app/views/projects/show.html.erb

```diff
@@ -66,7 +66,7 @@
                           </div>
                         </div>
                       <% end %>
-                      <%= link_to project_story_path(@project.id, story, format: "json"), method: "delete",  data: { confirm: 'Are you sure?', story_id: story.id }, class: "delete magenta", remote: true, title: "Delete" do %>
+                      <%= link_to project_story_path(@project.id, story), method: "delete",  data: { confirm: 'Are you sure?', story_id: story.id }, class: "delete magenta", remote: true, title: "Delete" do %>
```

Comment:
TIL!

Reply:
Also, instead of using a `format: "json"` to the named route (that generates the url with `.json` at the end) rails-usj allows a `type: :json` option for the helper that works along with the `remote: true` to do a json request by setting headers instead of url extension

https://www.ombulabs.com/blog/learning/javascript/behind-the-scenes-rails-ujs.html I added a comment about that feature here